### ISSUE TITLE
[Feature]  Support focus bracketing

### ISSUE THREAD

**Issue by cryptomilk** [ThibaudM/timelapse-sony#38]
Hi,

would it be possible to support focus backeting?

So either you select 3 or 5 focus points and the camera will shot them, or a start and an end and the steps between.

See e.g. https://github.com/obs1dium/FocusBracket

The A7M3 doesn't have applications. So the only way is to control it remote.

**Comment by cryptomilk** [comment]
I could write a PoC in python.

**Comment by retracked** [comment]
I don't think this would fit into a timelapse app time like this one. It is rather a matter for a new app.
